shell/pages/home.vue

```diff
@@ -40,9 +41,14 @@ export default {
 
   mixins: [PageHeaderActions],
 
-  fetch() {
-    this.$store.dispatch('management/findAll', { type: CAPI.RANCHER_CLUSTER });
-    this.$store.dispatch('management/findAll', { type: MANAGEMENT.CLUSTER });
+  async fetch() {
+    const res = await allHash({
+      rancherClusters: this.$store.dispatch('management/findAll', { type: CAPI.RANCHER_CLUSTER }),
+      mgmtClusters:    this.$store.dispatch('management/findAll', { type: MANAGEMENT.CLUSTER }),
+      psps:            this.$store.dispatch('management/request', { url: '/v1/policy.podsecuritypolicies' }),
```

Comment:
Banner should be on the cluster dashboard page for a cluster, not the home page

Reply:
Right - This banner only shows if the PSPs are in the local Rancher server cluster, so this banner assumes you're an admin who has access to the local cluster. But a cluster owner for a downstream cluster should be able to see the banner too.